### PR TITLE
Manage Staking and Validator Groups

### PR DESCRIPTION
**Pull request by mir-nawaz**
## Description
Manage to stake, add nominators, and validate. 

## Motivation and Context
Add new nominators and validate nodes. 

## How has this been tested?
Tested with local edgeware to test for all transactions.

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [x] no

### CODE REVIEW

**Comment by drewstone** [comment]
@mir-nawaz I'd rebase against staking-ui now that the merge from master has gone in. It looks like this carries a lot of extra work over that I don't want to have to review again.

**Comment by drewstone** [comment]
@raykyri this is a lot of UI work too so would be great to take a look.

File: client/scripts/controllers/chain/substrate/staking.ts [hunk 1]
```diff
@@ -105,6 +236,14 @@ class SubstrateStaking implements StorageModule {
   public get validatorCount(): Observable<SessionIndex> {
     return this._Chain.query((api: ApiRx) => api.query.staking.validatorCount());
   }
+  public get validatorsAddress(): Observable<string[]> {
```
[the remark this answers is not in the document]
Comment (mir-nawaz) [reply client/scripts/controllers/chain/substrate/staking.ts:239]: Yes, You are right. But it's not included in this feature. @iqraabdulrauf is working on it, So I'll leave a comment to integrate in the future.

**Review by drewstone — CHANGES_REQUESTED** [review]

File: client/scripts/controllers/chain/substrate/shared.ts [hunk 2]
```diff
@@ -406,6 +406,13 @@ class SubstrateChain implements IChainModule<SubstrateCoin, SubstrateAccount> {
           this._ss58Format = +ss58Format.unwrapOr(42);
           this._tokenDecimals = +tokenDecimals.unwrapOr(12);
```
[the remark this answers is not in the document]
Comment (mir-nawaz) [reply client/scripts/controllers/chain/substrate/shared.ts:407]: Not sure, this is from legacy code. Can't find DEFAULT_TOKEN_DECIMALS anywhere.

**Pull request merged**